standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/IMetaStoreClient.java

```diff
@@ -2814,13 +2814,22 @@ Function getFunction(String catName, String dbName, String funcName)
   ValidTxnList getValidTxns(long currentTxn) throws TException;
 
   /**
-   * Get a structure that details valid transactions.
+   * Get a structure that details valid write ids.
    * @param fullTableName full table name of format <db_name>.<table_name>
    * @return list of valid write ids for the given table
    * @throws TException
    */
   ValidWriteIdList getValidWriteIds(String fullTableName) throws TException;
 
+  /**
+   * Get a structure that details valid write ids.
+   * @param fullTableName full table name of format <db_name>.<table_name>
+   * @param writeId The write id to get the current txn
```

Comment:
Not necessarily current txn. Shall rephrase it as "Table's write id to get the corresponding txn id".

Reply:
ok